src/Akka.Streams.Kafka/Stages/Consumers/Abstract/SingleSourceStageLogic.cs

```diff
@@ -67,14 +67,24 @@ protected override IActorRef CreateConsumerActor()
                 ? new PartitionEventHandlers.Chain(autoSubscription.PartitionEventsHandler.Value, internalHandler)
                 : internalHandler;
 
+            IStatisticsHandler statisticsHandler = new StatisticsHandlers.Empty();
+            if (_subscription is IAutoSubscription autoSubscriptionForStatisticsHandler && autoSubscriptionForStatisticsHandler.StatisticsHandler.HasValue)
+            {
+                statisticsHandler = autoSubscriptionForStatisticsHandler.StatisticsHandler.Value;
+            }
+            else if (_subscription is IManualSubscription manualSubscriptionForStatisticsHandler && manualSubscriptionForStatisticsHandler.StatisticsHandler.HasValue)
+            {
+                statisticsHandler = manualSubscriptionForStatisticsHandler.StatisticsHandler.Value;
+            }
```

Comment:
If we would make new members to be part of `ISubscription` interface, I think this code could also be simpler - what do you think?